docs/CucumberSyntax.md

```diff
@@ -135,11 +131,33 @@ Note these steps work for asserting both integer and decimal data. There are no
 * _{field} contains anything but null_, executes the generator and asserts that _field_ has a value in every row (i.e. no `null`s)
 
 ### Cucumber test style guide
-* Tests should be written to validate one piece of logic only per test.
+* Each test should be specific to one requirement.
 * Tests should specify definite expected results rather than using "should include".
-* All tables should use pipe characters "|" to denote the edges of the table and the table should be sized according to the largest item in the table.
-* The indentation of cucumber tests should be:
->   * **Feature / Background / Scenario / Scenario outline** - 0 spaces
->   * **Given / When / Then** - 5 spaces
->   * **And** - 7 spaces
->   * **Tables / JSON snippets** - 2 spaces indented from the above statement
+* All tables should be padded to the width of the largest item.
+* All block-level indentation should be 2 spaces, as below: 
+
+```gherkin
+Feature: Tetris row scoring
+  Users score points by completing horizontal rows, which then disappear. The number of points
+  they receive is proportional to the number of rows they simultaneously complete.
+
+  Background:
+    Given a Tetris game is in progress
+
+  Scenario: The player scores a Tetris
```

Comment:
I get this conveys the meaning but its a bit of a weird departure, can we use something more indicative or simple here.

Reply:
I wouldn't want to use anything Helix-related in the documentation of the indentation style - it'd add risk of getting out of sync. How about something like:

```gherkin
Feature: ...
  ...

  Background:
    Given ...

  Scenario: ...
    Given ...:
      | ... |
      | ... |
      | ... |
    And ...:
      """
      """
    When ...
    Then ... 
    And ...
```